hooks/check_commit_msg.py

```diff
@@ -23,10 +23,11 @@ def main(*args):
         data = FILE.read()
         msg = data.split('\n')[0]
         keyword = msg.split(':', maxsplit=1)[0]
-        if keyword not in keywords:
+        all_keyword = keywords + [s.capitalize() for s in keywords]
```

Comment:
should we all add all caps ? 
so then all these will be Valid 
`feat`, `Feat`, `FEAT`

Reply:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/3949397/180893306-8846f0f9-fb24-4520-8a61-796ce958ef82.png">
not all , only 'Feat' and 'feat'

Reply:
No I'm asking if we should also add FEAT

Reply:
or do we specify the first character in uppercase?

Reply:
Ok then add FEAT to valid keyword,
So we have feat, Feat, FEAT all these three are valid